src/Servers/Kestrel/Core/src/Internal/Http/HttpProtocol.Generated.cs

```diff
@@ -104,33 +106,50 @@ private object ExtraFeatureGet(Type key)
             return null;
         }
 
-        private void ExtraFeatureSet(Type key, object value)
+        private void ExtraFeatureSet(Type key, object? value)
```

Comment:
Previously calling ExtraFeatureSet with a null value would leave a null entry in MaybeExtra. These entries would still be returned when foreaching over feature collection.

Now null extra features are completely removed.